content/en/docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -25,10 +25,7 @@ include debugging utilities like a shell.
 
 {{% capture prerequisites %}}
 
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
-
-Process Namespace Sharing is a **beta** feature that is enabled by default. It
-may be disabled by setting `--feature-gates=PodShareProcessNamespace=false`.
+{{< include "task-tutorial-prereqs.md" >}}
```

Comment:
Why omit the `{{< version-check >}}` shortcode?

Reply:
Just assumed it was on the list of things to remove for GA. I can add it back.

The generated text is pretty misleading, though:

> Your Kubernetes server must be at or later than version v1.10. To check the version, enter kubectl version.

In fact if your Kubernetes server is at version v1.10 and you haven't specifically enabled the feature, then the task will fail.